diff --git a/cosypose/evaluation/meters/pose_meters.py b/cosypose/evaluation/meters/pose_meters.py
index 1181d5e..3028f4e 100644
--- a/cosypose/evaluation/meters/pose_meters.py
+++ b/cosypose/evaluation/meters/pose_meters.py
@@ -197,7 +197,7 @@ class PoseErrorMeter(Meter):
 
         matches['obj_diameter'] = 'match_id', [self.mesh_db.infos[k.item()]['diameter_m'] for k in matches['label']]
         matches['norm'] = 'match_id', errors_norm
-        matches['0.1d'] = 'match_id', errors_norm < 0.1 * matches['obj_diameter']
+        matches['0.1d'] = 'match_id', errors_norm < 0.1 * matches['obj_diameter'].data
         matches['xyz'] = ('match_id', 'dim3'), errors_xyz
         matches['TCO_xyz'] = ('match_id', 'dim3'), errors_TCO_xyz
         matches['TCO_norm'] = 'match_id', errors_TCO_norm
@@ -221,7 +221,7 @@ class PoseErrorMeter(Meter):
 
         preds_match_merge = xr_merge(preds, matches, on=group_keys+['pred_inst_id'],
                                      dim1='pred_id', dim2='match_id', fill_value=fill_values)
-        preds['0.1d'] = 'pred_id', preds_match_merge['0.1d']
+        preds['0.1d'] = 'pred_id', preds_match_merge['0.1d'].data
 
         self.datas['gt_df'].append(gt)
         self.datas['pred_df'].append(preds)
