diff --git a/cosypose/evaluation/meters/pose_meters.py b/cosypose/evaluation/meters/pose_meters.py
index 1181d5e..c4a6dab 100644
--- a/cosypose/evaluation/meters/pose_meters.py
+++ b/cosypose/evaluation/meters/pose_meters.py
@@ -275,7 +275,9 @@ class PoseErrorMeter(Meter):
 
         df = pred_df[['label', valid_k, 'score']].to_dataframe().set_index(['label'])
         for label, label_n_gt in n_gts.items():
-            if df.index.contains(label):
+            print("df.index type is: ", type(df.index))
+            #if df.index.contains(label):  https://pandas.pydata.org/pandas-docs/version/0.25/reference/api/pandas.Index.contains.html
+            if label in df.index:
                 label_df = df.loc[[label]]
                 if label_df[valid_k].sum() > 0:
                     ap, label_df = compute_ap(label_df, label_n_gt)
